src/components/VMenu/VMenu.js

```diff
@@ -190,7 +190,7 @@ export default {
     const data = {
       staticClass: 'v-menu',
       style: {
```

Comment:
is the style here required at all?

Reply:
It's just required if the $slots.activator is being used. But it's not printed in the actual markup of the code.

Reply:
Then I'd add it only if slot is used

Reply:
Yep, I tried removing the style property and then this:
```
if (this.$slots.activator) {
      data['style']['display'] = this.fullWidth ? 'block' : 'inline-block'
}
```
But tests failed. I searched if I was doing something wrong but couldn't find any explanation. Have any suggestion?

Reply:
`data.style = {...}`